src/main/java/io/cdap/plugin/format/DBTypes.java

```diff
@@ -90,6 +90,13 @@ private static Schema getSchema(ResultSetMetaData metadata, int column) throws S
       case Types.DECIMAL:
         int precision = metadata.getPrecision(column);
         int scale = metadata.getScale(column);
+        String typeName = metadata.getColumnTypeName(column);
+        // decimal type with precision 0 is not supported
+        if (precision == 0) {
```

Comment:
IIRC, Oracle pure Number without precision or scale will have precision 0. Could you test it ?
https://docs.oracle.com/cd/B28359_01/server.111/b28318/datatype.htm#CNCPT1832

Reply:
while we don't have a specific Oracle plugin for multi-table, maybe we need to discuss with PM about what we should do here.

Reply:
Yeah, any data type where precision and scale is not defined in the schema, oracle jdbc will have this limitation. Yeah, I also thought the same because it is generic plugin.